openpype/client/operations.py

```diff
@@ -708,7 +707,14 @@ def delete_entity(self, project_name, entity_type, entity_id):
         return operation
 
 
-def create_project(project_name, project_code, library_project=False):
+def create_project(
```

Comment:
Please don't change this function arguments, it would break our AYON preparation + the selection of attributes, that can be passed, is very limited. All manager are handling this changes by updating existing project.

Reply:
@iLLiCiTiT Would you please give me a hint on how can I properly update a newly created project? 
I tried [here ](https://github.com/ynput/OpenPype/pull/5075/commits/7dae81126d60ef5d71aa15a068a9236952d718bb) to use op.session, and now the project creates now with new settings, but all other elements such as frame start etc are empty.
![image](https://github.com/ynput/OpenPype/assets/11698866/2df68f51-29f0-41dd-9d96-52c43a8f4d64)

Reply:
Oh, I guess I have figured it out. Please check if this is a correct approach: https://github.com/ynput/OpenPype/pull/5075/commits/57cf805a7b8e05f507bf0f05657b78177afdabc7